packages/api/db/migration/20231010170020_create_finance_report_table.js

```diff
@@ -0,0 +1,46 @@
+/*
+ *  Copyright (c) 2023 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+export const up = async function (knex) {
+  await knex.schema.createTable('finance_report', (table) => {
+    table.increments('finance_report_id').primary();
+    table.uuid('farm_id').references('farm_id').inTable('farm').defaultTo(null);
```

Comment:
Should this be not nullable too, or is there a way a user could generate a report not belonging to any farm?

Reply:
No, it should always refer to a farm! I'll fix that, thanks 🙂